static/js/actions/index.js

```diff
@@ -57,10 +61,20 @@ export const updateProfile = profile => ({
   type: UPDATE_PROFILE,
   payload: { profile }
 });
+export const openEducationForm = (level, index) => ({
+  type: SHOW_EDUCATION_FORM_DIALOG,
+  payload: { level, index }
+});
+export const closeEducationForm = () => ({type: HIDE_EDUCATION_FORM_DIALOG});
 
 export const startProfileEdit = () => ({ type: START_PROFILE_EDIT });
 export const clearProfileEdit = () => ({ type: CLEAR_PROFILE_EDIT });
 
+export const toggleEducationLevel = newState => ({
```

Comment:
Can you rename `newState` to `educationLevels` here? You will also need to change the reducer accordingly